addons/web_editor/static/src/js/editor/rte.summernote.js

```diff
@@ -30,11 +30,15 @@ var tplButton = renderer.getTemplate().button;
 var tplIconButton = renderer.getTemplate().iconButton;
 var tplDropdown = renderer.getTemplate().dropdown;
 
-var applyColor = function (target, eventName, color) {
+const processAndApplyColor = function (target, eventName, color) {
+    if (!color) {
+        color = 'inherit';
```

Comment:
Why forcing inherit? (was not the case before, right?)

Reply:
it was the case in color reset. Now when the color picked is an empty string it is similar to a color reset.